### PR TITLE
Add the consumers verification steps after a provider node replacement to the relevant MS node tests

### PR DESCRIPTION
**Pull request by yitzhak12**

### CODE REVIEW

**Comment by yitzhak12** [comment]
The pr validation job passed: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/1064/.

File: ocs_ci/ocs/resources/pod.py [hunk 1]
```diff
@@ -2946,3 +2946,52 @@ def restart_pods_in_statuses(
...
+
+def check_ceph_cmd_execute_successfully():
+    """
+    Check that a Ceph command executes successfully
+
+    Returns:
+        bool: True, if the Ceph command executed successfully. False, otherwise
+
+    """
+    # Import here to avoid circular loop
```
Comment (fbalak) [inline ocs_ci/ocs/resources/pod.py:2981]: Why does having those modules imported properly at the beginning of the file cause circular import?

Comment (yitzhak12) [reply ocs_ci/ocs/resources/pod.py:2981]: These are the errors I get when putting them at the beginning of the file:

> File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/framework/pytest_customization/ocscilib.py", line 31, in <module>
    from ocs_ci.ocs.cluster import check_clusters
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/cluster.py", line 22, in <module>
    import ocs_ci.ocs.resources.pod as pod
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/resources/pod.py", line 18, in <module>
    from ocs_ci.ocs.cluster import is_ms_consumer_cluster
ImportError: Error importing plugin "ocs_ci.framework.pytest_customization.ocscilib": cannot import name 'is_ms_consumer_cluster' from partially initialized module 'ocs_ci.ocs.cluster' (most likely due to a circular import) (/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/cluster.py)

> File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/framework/pytest_customization/ocscilib.py", line 31, in <module>
    from ocs_ci.ocs.cluster import check_clusters
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/cluster.py", line 22, in <module>
    import ocs_ci.ocs.resources.pod as pod
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/resources/pod.py", line 18, in <module>
    from ocs_ci.ocs.managedservice import patch_consumer_toolbox
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/managedservice.py", line 11, in <module>
    from ocs_ci.ocs.resources.pod import get_ceph_tools_pod, get_pods_having_label, Pod
ImportError: Error importing plugin "ocs_ci.framework.pytest_customization.ocscilib": cannot import name 'get_ceph_tools_pod' from partially initialized module 'ocs_ci.ocs.resources.pod' (most likely due to a circular import) (/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/resources/pod.py)

**Comment by yitzhak12** [comment]
The pr validation passed: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/1077/. There is a small error in the teardown that doesn't relate to the test:

> 2022-11-27 15:08:56  INTERNALERROR>   File "/home/jenkins/workspace/qe-odf-multicluster/ocs-ci/venv/lib64/python3.8/site-packages/pytest_logger/plugin.py", line 119, in pytest_runtest_makereport
2022-11-27 15:08:56  INTERNALERROR>     logger.on_makereport()
2022-11-27 15:08:56  INTERNALERROR>   File "/home/jenkins/workspace/qe-odf-multicluster/ocs-ci/venv/lib64/python3.8/site-packages/pytest_logger/plugin.py", line 142, in on_makereport
2022-11-27 15:08:56  INTERNALERROR>     _disable(self.handlers)
2022-11-27 15:08:56  INTERNALERROR>   File "/home/jenkins/workspace/qe-odf-multicluster/ocs-ci/venv/lib64/python3.8/site-packages/pytest_logger/plugin.py", line 383, in _disable
2022-11-27 15:08:56  INTERNALERROR>     hdlr.close()
2022-11-27 15:08:56  INTERNALERROR>   File "/usr/lib64/python3.8/logging/__init__.py", line 1163, in close
2022-11-27 15:08:56  INTERNALERROR>     stream.close()
2022-11-27 15:08:56  INTERNALERROR> OSError: [Errno 5] Input/output error

File: ocs_ci/ocs/node.py [hunk 2]
```diff
@@ -2535,6 +2539,18 @@ def consumer_verification_steps_after_provider_node_replacement():
             )
             return False
 
+    log.info("Check if we can execute a ceph command from the consumer")
+    if not pod.wait_for_ceph_cmd_execute_successfully(timeout=300):
+        log.info("Try to restart the ocs-operator pod")
```
Comment (jilju) [inline ocs_ci/ocs/node.py:2544]: Please mention the bug ID. This step can be removed after the bug fix.

Comment (yitzhak12) [reply ocs_ci/ocs/node.py:2544]: Yes, that will do.

**Comment by yitzhak12** [comment]
The pr validation passed: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/1089/.

**Comment by yitzhak12** [comment]
The pr validation passed: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/1091/console.

**Review by fbalak — APPROVED** [review]

**Review by jilju — APPROVED** [review]

**Pull request merged**